packages/ozone/src/mod-service/index.ts

```diff
@@ -731,7 +750,14 @@ export class ModerationService {
       )
     }
 
-    const { ref } = this.db.db.dynamic
+    if (tags.length) {
+      builder = builder.where(
+        sql<string>`${ref(
```

Comment:
This wont evaluate to a string, but the type also doesn't need to be preserved here anyway so just nixing it.
```suggestion
        sql`${ref(
```